jhdf/src/main/java/io/jhdf/HdfFile.java

```diff
@@ -272,4 +272,11 @@ public boolean isLinkCreationOrderTracked() {
 		return rootGroup.isLinkCreationOrderTracked();
 	}
 
+	/**
+	 * @return the underlying {@link HdfFileChannel}
+	 */
+	public HdfFileChannel getHdfChannel() {
```

Comment:
Do you have a use case for the `HdfFileChannel`? Or do you just want it to access the underlying Java `FileChannel`? If so I would propose we just make this method return the Java `FileChannel` directly?

Reply:
We use this for creating global heaps from an address
`GlobalHeap heap = new GlobalHeap(hdfFile.getHdfChannel(), heapAddress);`
or to get an ObjectHeader for an address like in the first comment.

Reply:
This seems reasonable